docs/docs/modules.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: KAMI Blue Modules
+description: KAMI Blue Module List
+---
+
+## KAMI Blue Modules
+
+<details>
+    <summary>Capes</summary>
+    <ul>
+        <li>Capes</li>
+        <li>LayerCape</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Combat</summary>
+    <ul>
+        <li>AntiDeathScreen</li>
+        <li>AntiFriendHit</li>
+        <li>Aura</li>
+        <li>Auto32k</li>
+        <li>AutoExp</li>
+        <li>AutoFeetPlace</li>
+        <li>AutoLog</li>
+        <li>AutoReplenish</li>
+        <li>AutoTotem</li>
+        <li>AutoTrap</li>
+        <li>BowSpam</li>
+        <li>Criticals</li>
+        <li>CrystalAura</li>
+        <li>Fastuse</li>
+        <li>Pull32k</li>
+        <li>Surround</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Experimental</summary>
+    <ul>
+        <li>GUIColour</li>
+    </ul>
+</details>
+
+<details>
+    <summary>GUI</summary>
+    <ul>
+        <li>ArmourHide</li>
+        <li>CleanGUI</li>
+        <li>InfoOverlay</li>
+        <li>InventoryViewer</li>
+        <li>Zoom</li>
+        <li>GUIColour</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Hidden</summary>
+    <ul>
+        <li>Teleport</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Misc</summary>
+    <ul>
+        <li>AntiAFK</li>
+        <li>AntiChatSpam</li>
+        <li>AntiChunkBan</li>
+        <li>AntiWeather</li>
+        <li>AutoFish</li>
+        <li>AutoQMain</li>
+        <li>AutoReconnect</li>
+        <li>AutoRespawn</li>
+        <li>AutoSpawner</li>
+        <li>AutoTPA</li>
+        <li>AutoTool</li>
+        <li>BookCrash</li>
+        <li>CameraClip</li>
+        <li>ChatEncryption</li>
+        <li>ColourSign</li>
+        <li>ConsoleSpam</li>
+        <li>CustomChat</li>
+        <li>DiscordSettings</li>
+        <li>FakeGamemode</li>
+        <li>FakeVanilla</li>
+        <li>FormatChat</li>
+        <li>NoEntityTrace</li>
+        <li>NoPacketKick</li>
+        <li>NoSoundLag</li>
+        <li>PortalChat</li>
+        <li>SkinFlicker</li>
+        <li>VisualRange</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Movement</summary>
+    <ul>
+        <li>AntiHunger</li>
+        <li>AutoJump</li>
+        <li>AutoWalk</li>
+        <li>ElytraFlight</li>
+        <li>EntitySpeed</li>
+        <li>Flight</li>
+        <li>IceSpeed</li>
+        <li>Jesus</li>
+        <li>NoSlowDown</li>
+        <li>SafeWalk</li>
+        <li>Strafe</li>
+        <li>TimerSpeed</li>
+        <li>Velocity</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Player</summary>
+    <ul>
+        <li>AntiChunkLoadPatch</li>
+        <li>AntiForceLook</li>
+        <li>AutoArmour</li>
+        <li>AutoEat</li>
+        <li>Blink</li>
+        <li>Fastbreak</li>
+        <li>Freecam</li>
+        <li>NoBreakAnimation</li>
+        <li>NoFall</li>
+        <li>NoSwing</li>
+        <li>PitchLock</li>
+        <li>PortalGodMode</li>
+        <li>Scaffold</li>
+        <li>Timer.java</li>
```

Comment:
Why do these have .java

Reply:
oops, must have made a mistake when scraping the directory structure